### ISSUE TITLE
No Password Error

### ISSUE THREAD

**Issue by wasdee** [heypoom/spending-visualizer#1]
**Browser**: Firefox Latest
**Expect**: a prompt for password
**Experience**: Silent Error
Console shown
```
Uncaught (in promise) 
Object { message: "No password given", name: "PasswordException", code: 1, stack: "__webpack_modules__</d<@https://spending.poom.dev/assets/index-89450f41.js:1:35080\n__webpack_modules__<@https://spending.poom.dev/assets/index-89450f41.js:1:35108\n__w_pdfjs_require__@https://spending.poom.dev/assets/index-89450f41.js:6:11626\n@https://spending.poom.dev/assets/index-89450f41.js:6:15945\n@https://spending.poom.dev/assets/index-89450f41.js:6:16181\n@https://spending.poom.dev/assets/index-89450f41.js:6:16205\n@https://spending.poom.dev/assets/index-89450f41.js:1:27721\n@https://spending.poom.dev/assets/index-89450f41.js:1:27726\n@https://spending.poom.dev/assets/index-89450f41.js:6:16210\n" }
index-89450f41.js:6:18415
```

**Comment by wasdee** [comment]
if you need file pls ask me privately via keybase

**Comment by wasdee** [comment]
https://www.loom.com/share/83a4ad0feeee426e8c420048716da5df

**Comment by heypoom** [comment]
@wasdee yes krub, currently you need to decode the statement file in other PDF program as I haven't added the Password Input field yet. Planning on fixing this today at 10PM in the livestream!

**Comment by heypoom** [comment]
Solved by @GGolfz with #11. Awesome!

**Comment by heypoom** [comment]
@all-contributors add @wasdee for bug
